technical-doc/api.rst

```diff
@@ -15,34 +15,1170 @@ Massa JSON-RPC API is splitted in two parts :
 
 **Public API**: used for blockchain interactions. Default port: 33035 e.g. http://localhost:33035
 
-Find the complete Massa `OpenRPC <https://spec.open-rpc.org/>`_  specification `here <https://raw.githubusercontent.com/massalabs/massa/main/docs/technical-doc/openrpc.json>`_.
+Find the complete Massa `OpenRPC <https://spec.open-rpc.org/>`_  specification `here <https://raw.githubusercontent.com/massalabs/massa/main/massa-node/base_config/openrpc.json>`_.
 
+Integrations
+============
+
+**JavaScript**: use `massa-web3.js <https://github.com/massalabs/massa-web3>`_.
+
+**Smart contracts**: use `massa-as-sdk <https://github.com/massalabs/massa-as-sdk>`_.
```

Comment:
massa-as-sdk does not talk to the JSON-RPC-API, right ?

Reply:
Nope, It's a section to redirect users to the right project, this why we choose the world `use`

Reply:
Yes, but it does not really seem relevant to redirect users to `massa-as-sdk` as it's not related to the API.

Reply:
We could remove it, no problem

Reply:
I would remove it then.